dt-core/release-notifications.php

```diff
@@ -71,8 +124,35 @@ function dt_release_modal() {
 }
 add_action( 'wp_head', 'dt_release_modal' );
 
+function dt_get_plugins_news_links() {
+    $plugin_news_url = 'https://disciple.tools/news-categories/dt-plugin-releases/feed/';
```

Comment:
@prykon is there a feed for all the news except the theme news?

Reply:
WordPress has feeds for categories and the generic one. I never heard of a feed sans a specific category, but I'll double check.

Reply:
@prykon Or we create a type on the https://disciple.tools/newsb that is just "other"